src/components/text/index.tsx

```diff
@@ -1,14 +1,38 @@
 import * as React from 'react';
 import { styled } from 'linaria/react';
-import { sizes, colors } from 'styles';
+import { sizes } from 'styles';
 import { useTheme } from 'utils/hooks';
-export interface TextProps extends React.HTMLAttributes<HTMLSpanElement> {
+
+export interface TextProps extends React.HTMLAttributes<HTMLElement> {
   color?: any;
   weight?: any;
   size?: any;
+  component?: string;
```

Comment:
shouldn't it be 'p' | 'span'?

Reply:
for now yes. idk atm it will be changed in the future or not. 
but it more clear if describe it like "p" | "span" atm